plugins/telemetry/telemetry_impl.h

```diff
@@ -30,7 +30,7 @@ class TelemetryImpl : public PluginImplBase {
     Telemetry::Result set_rate_position(double rate_hz);
     Telemetry::Result set_rate_home_position(double rate_hz);
     Telemetry::Result set_rate_in_air(double rate_hz);
-    Telemetry::Result set_rate_status_text(double rate_hz);
+    // Telemetry::Result set_rate_status_text(double rate_hz); Anotacao
```

Comment:
I think you can delete these things.

Reply:
Hm, I did. Not sure why it's still showing here. 🤔

Reply:
Now it seems to show all my old "Anotacao" messages. Will check how to fix it.